src/index.html

```diff
@@ -7,12 +7,48 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Calendar</title>
+    <link
+      rel="stylesheet"
+      href="styles/reset.css"
+    />
     <link
       rel="stylesheet"
       href="styles/index.scss"
```

Comment:
The href attribute in the link tag is pointing to a SCSS file. Browsers cannot directly interpret SCSS files. You should compile your SCSS into CSS and link to the resulting CSS file instead.